x-pack/plugins/uptime/public/components/overview/alerts/alerts_containers/alert_monitor_status.tsx

```diff
@@ -27,19 +29,40 @@ export const AlertMonitorStatus: React.FC<Props> = ({
   numTimes,
   setAlertParams,
   timerange,
+  alertParams,
 }) => {
-  const { filters, locations } = useSelector(selectMonitorStatusAlert);
-  const searchText = useSelector(searchTextSelector);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    // example output: `now-13h`
+    const dateRangeStart = `now-${alertParams?.timerangeCount ?? 15}${
+      alertParams?.timerangeUnit ?? 'd'
+    }`;
```

Comment:
i don't understand this part, why is it using rimeRangeCount for date range filter? shouldn't it be default to now-15m or maybe empty filter.

Reply:
There was never any conversation about this; my initial thought was it would be best to use the date span reflected in the context of the alert. A blank date range would probably be best; the date range is required today, so we'd need to modify the server to make that work.

Perhaps for now we should hardcode `now-15m` and update it in a follow-up, as it's not a bug IMO.

Reply:
i would say let's default it to now-1d, that's safe time period. now-15m is too short.

Reply:
Addressed in https://github.com/elastic/kibana/pull/68005/commits/e59dc77cc3bbd63117edbd0e354bbdc8f6d3872d.